File: Introduction to Computer Science/Week 1/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    int counter = 0;
    float f;
    do
    {
        f = get_float("Change owed: ");
    }

    while (f < 0);

    int cents = round(f * 100);

    while(cents >= 25){
        cents -= 25;
        counter++;
    }

     while(cents >= 10){
        cents -= 10;
        counter++;
    }

     while(cents >= 5){
        cents -= 5;
        counter++;
    }

     while(cents >= 1){
        cents -= 1;
        counter++;
    }

    printf("%i\n", counter);
}

File: Introduction to Computer Science/Week 6/pset6/cash/cash.py
from cs50 import get_float

while True:
    try:
        h = int(get_float("Change owed: ") * 100)
        if 0 <= h:
            break
    except ValueError:
        continue

count = 0


for val in [25, 10, 5, 1]:
    while h >= val:
        h -= val
        count += 1
print(count)